tutorials/kafkaexactlyonce/src/main/java/com/example/Application.java

```diff
@@ -1,5 +1,20 @@
 /**
```

Comment:
This license text applies to code submitted to ASF; to merely apply the Apache license, use the text here:
http://www.apache.org/licenses/LICENSE-2.0.html#apply